app/Jobs/ProcessDemoTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use App\Models\DemoTest;
+use App\Models\DemoTestInquiry;
+use Illuminate\Support\Facades\Log;
+
+class ProcessDemoTest implements ShouldQueue
```

Comment:
5. “Dispatcher” job must dispatch a separate (queueable) individual new job per each object inside the “demo_test_inquiry.payload”,

Reply:
created new individual job for payload pices and batched them